terasoluna-gfw-parent/pom.xml

```diff
@@ -337,6 +337,19 @@
       </dependency>
       <!-- == End MyBatis == -->
 
+      <!-- == Begin Hibernate == -->
+      <dependency>
+        <groupId>org.hibernate</groupId>
+        <artifactId>hibernate-core</artifactId>
+        <version>${org.hibernate.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.hibernate</groupId>
+        <artifactId>hibernate-entitymanager</artifactId>
+        <version>${org.hibernate.version}</version>
+      </dependency>
+      <!-- == End Hibernate == -->
```

Comment:
By upgrading hibernate-core

- [ ] Transitive dependencies are downgraded by Spring Boot. (Check whether manage version or not.)
* byte-buddy 1.9.11 -> 1.9.7

- [ ] Transitive dependency added. (Need to reflect in each application, guidelines.)
* jaxb-runtime 2.3.1

Reply:
byte-buddy has not been upgraded as it has been confirmed that there are no problems with downgrading.